pkg/kpackcompat/clientset.go

```diff
@@ -0,0 +1,90 @@
+package kpackcompat
+
+import (
+	"fmt"
+
+	"github.com/pivotal/kpack/pkg/apis/build"
+	"github.com/pivotal/kpack/pkg/client/clientset/versioned/typed/build/v1alpha2"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/discovery"
+
+	// load credential helpers
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/util/flowcontrol"
+)
+
+const (
+	KpackGroupVersionV1alpha2 = build.GroupName + "/v1alpha2"
+)
+
+type ClientsetInterface interface {
```

Comment:
Is defining our own ClientSetInterface necessary?

Reply:
This limits where our compat layer can be used as any other kpack golang logic would depend on the official kpack ClientSet.